### PR TITLE
release note: v10.5.1

### PR DESCRIPTION
**Pull request by fredmaggiowski**
## Pull Request Type

- [ ] Bugfix
- [ ] Documentation content changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../CONTRIBUTING.md#content-checks))
- [ ] No sensible content has been committed

### CODE REVIEW

File: docs/release-notes/platform_10-5-1_releasenotes.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+id: v10.5.1
+title: Version 10.5.1 Release Notes
+sidebar_label: v10.5.1
+image: "img/release-note-link-preview.png"
+---
+
+_March 02, 2023_
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-5-1_releasenotes.md:8]:
```suggestion
_March 07, 2023_
```

File: docs/release-notes/platform_10-5-1_releasenotes.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+### Bug Fix
+
+#### Editor does not use custom tags
+Output of fields with format `editor` do not use custom `quill` classes in form components. Refer to [this issue](https://github.com/zenoamaro/react-quill/issues/553) for further details
+
+
+## How to update your Console
+
+For on-premise Console installations, please contact your Mia Platform referent to know how to use the `Helm chart version X.X.X`.
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-5-1_releasenotes.md:152]: da aggiornare la chart helm

Comment (davidebianchi) [reply docs/release-notes/platform_10-5-1_releasenotes.md:155]: In english please 😄

Comment (fredmaggiowski) [reply docs/release-notes/platform_10-5-1_releasenotes.md:155]:
```suggestion
For on-premise Console installations, please contact your Mia Platform referent to know how to use the `Helm chart version 8.3.7`.
```

**Pull request merged**